cdc/sink/codec/common/config.go

```diff
@@ -41,8 +41,12 @@ type Config struct {
 	AvroDecimalHandlingMode        string
 	AvroBigintUnsignedHandlingMode string
 
-	// csv only
-	CSVConfig *config.CSVConfig
+	// for sinking to cloud storage
```

Comment:
For me, the idea of `Delimiter` / `Quote` and `Terminator` can not be easily distinguished. How about reorder the declaration of them, and add some comments about the usage and difference?